src/rust/plugin-registry/src/static_files/hax_docker_analyzer.nomad

```diff
@@ -95,21 +101,37 @@ job "grapl-plugin" {
         sidecar_service {
           proxy {
             upstreams {
-              destination_name = "plugin-work-queue"
-              # port unique but arbitrary - https://github.com/hashicorp/nomad/issues/7135
-              local_bind_port = 1000
+              destination_name      = "plugin-work-queue"
+              destination_namespace = "default"
+              local_bind_port       = 1000
             }
 
             upstreams {
-              destination_name = "plugin-${var.plugin_id}"
-              # port unique but arbitrary - https://github.com/hashicorp/nomad/issues/7135
-              local_bind_port = 1001
+              destination_name      = "plugin"
```

Comment:
would it be possible to call this "analyzer" instead of "plugin"? It would be nice to make the corresponding name "generator" for the generator plugins as well

Reply:
done